src/test/java/tests/TurboIssueEventTests.java

```diff
@@ -17,11 +17,11 @@
 
     private static TurboIssueEvent createLabelUpdateEvent(
             String userName, IssueEventType eventType,
-            GregorianCalendar time, String labelName) {
+            GregorianCalendar time, String labelName, String labelColour) {
```

Comment:
I don't think GregorianCalendar makes much of a difference; we just need the Date object, anyway.